### PR TITLE
docs: Update ignore.md

### PR DESCRIPTION
**Pull request by openscript**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

**Comment by amareshsm** [comment]
@openscript  Please sign the [CLA](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/19114526/11061773/18963/#/?version=2) 

Also, please update the PR description to clarify the changes being made in this pull request.

File: docs/src/use/configure/ignore.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ You can configure ESLint to ignore certain files and directories while linting b
 
 ## Ignoring Files
 
+::: tip
+Be aware that `ignores` need to be in it's own object. Combining with other configuration, such as `files` doesn't work!
```
Comment (amareshsm) [inline docs/src/use/configure/ignore.md:23]:
```suggestion
Be aware that `ignores` needs to be in its own object. Combining with other configurations, such as `files` doesn't work!
```

**Comment by nzakas** [comment]
This is just repeating the same content that occurs in the following paragraph. I don't see the purpose of this.

**Comment by pfroud** [comment]
A lot of people have expressed frustration about the behavior of `ignores`: https://github.com/eslint/eslint/discussions/18304

I think emphasizing how to add a global ignore would be helpful.

**Pull request closed without merging**